src/telemetry/telemetryService.ts

```diff
@@ -477,14 +517,14 @@ export default class TelemetryService {
       command,
       error_code: errorCode,
       error_name: errorName,
-    });
+    } as ParticipantResponseFailedProperties);
```

Comment:
[non-blocking]

```suggestion
    } satisfies ParticipantResponseFailedProperties);
```

I think we can do this now, our TS version and tooling seems to be fine with it (and it's generally nicer since it avoids the behavior of `as` that allows type coercions even in case of mismatches)